### ISSUE TITLE
matoya's_relict timeline intended bug(?)

### ISSUE THREAD

**Issue by Jaehyuk-Lee** [quisquous/cactbot#3169]
## Question

What would you like to ask?

The "duration" timeline is not removed well. It's already finished, but I guess it can be that long if party don't kill the wind thing. I don't know how to fix this in an appropriate way.
![ezgif com-crop](https://user-images.githubusercontent.com/4695371/122921745-da41be80-d39d-11eb-80d9-6b5cea6f5e09.gif)

`3096.6 "Huff And Puff" sync /:591A:Mother Porxie/ window 96.6,5 duration 49.70`
Just shortening the duration seems bad.

**Comment by valarnin** [comment]
In the timeline, this:

```
3096.6 "Huff And Puff" sync /:591A:Mother Porxie/ window 96.6,5 duration 49.70
3101.6 "Buffet" sync /:Aeolian Cave Sprite:5926:/
3107.3 "Buffet" sync /:Aeolian Cave Sprite:5926:/
3112.8 "Buffet" sync /:Aeolian Cave Sprite:5926:/
3118.4 "Buffet" sync /:Aeolian Cave Sprite:5926:/
3124.0 "Buffet" sync /:Aeolian Cave Sprite:5926:/
3129.6 "Buffet" sync /:Aeolian Cave Sprite:5926:/
3130.3 "Huff And Puff Enrage?" sync /:Mother Porxie:591B:/ window 50,5

3132.3 "--targetable--"
# Loop
3132.4 "Medium Rear" sync /:Mother Porxie:591D:/
```

There's an overlap between the first line and the `--targetable--` of roughly 14 seconds. Either enrage time is wrong, or duration time is wrong?

**Comment by Jaehyuk-Lee** [comment]
The Second line, "불쏘시개" is "Open Flame" from 
`3148.5 "Open Flame" sync /:Mother Porxie:5922:/`
So that `Huff And Puff` lasting until about 3146.0.

It looks like shortening duration to `34.3` would fix this?

**Comment by MaikoTan** [comment]
It worked normally when I created this file, I guess because of the higher dps right now, the Sprite died too early which cause the issue?
"Huff And Puff" is the full-field AoE (with arrows), right?

**Comment by Jaehyuk-Lee** [comment]
> "Huff And Puff" is the full-field AoE (with arrows), right?

I'm not sure about that.

**Comment by MaikoTan** [comment]
I think if the duration of "Huff And Puff" is dynamical depending on when the Spirit dies, is it make sense to just remove the duration entirely?

**Comment by valarnin** [comment]
The problem isn't with the duration, the problem is with the rest of the timeline after that mechanic. It overlaps, so the duration is going to be displayed.

You could solve the issue with the following:

```
3096.6 "Huff And Puff" sync /:591A:Mother Porxie/ window 96.6,5 duration 49.70
3101.6 "Buffet" sync /:Aeolian Cave Sprite:5926:/
3107.3 "Buffet" sync /:Aeolian Cave Sprite:5926:/
3112.8 "Buffet" sync /:Aeolian Cave Sprite:5926:/
3118.4 "Buffet" sync /:Aeolian Cave Sprite:5926:/
3124.0 "Buffet" sync /:Aeolian Cave Sprite:5926:/
3129.6 "Buffet" sync /:Aeolian Cave Sprite:5926:/
3135.2 "Buffet" sync /:Aeolian Cave Sprite:5926:/
3140.8 "Buffet" sync /:Aeolian Cave Sprite:5926:/
3144.3 "Huff And Puff Enrage?" sync /:Mother Porxie:591B:/ window 50,5

3146.3 "--targetable--"
# Loop
3146.4 "Medium Rear" sync /:Mother Porxie:591D:/
3162.5 "Open Flame" sync /:Mother Porxie:5922:/
3174.6 "Minced Meat" sync /:Mother Porxie:5911:/
3186.7 "Tender Loin" sync /:Mother Porxie:5913:/
3200.8 "Tender Loin" sync /:Mother Porxie:5913:/
3221.1 "Huff And Puff" sync /:Mother Porxie:5919:/

3223.2 "Medium Rear" sync /:Mother Porxie:591D:/
3239.3 "Open Flame" sync /:Mother Porxie:5922:/
3251.5 "Minced Meat" sync /:Mother Porxie:5911:/ jump 3160.6
3263.6 "Tender Loin" # sync /:Mother Porxie:5913:/
3277.7 "Tender Loin" # sync /:Mother Porxie:5913:/
3295.9 "Huff And Puff" # sync /:Mother Porxie:5919:/"
```

That's two extra `Buffet` casts, then enrage and all events afterwards are adjusted up by 14 seconds.

**Comment by MaikoTan** [comment]
Ah, I see!

**Comment by Jaehyuk-Lee** [comment]
@valarnin could you please PR that? I'm not sure about this part, I don't have enrage experience. So, I can't answer if there's any question about it.

**Comment by valarnin** [comment]
I've never seen past the 2nd `Buffet` cast myself. You'd need to go in as a group of four and just sit there to verify the behavior, even a solo healer can burn the add down in 4-5 casts of `Buffet`.

**Comment by Jaehyuk-Lee** [comment]
@valarnin I don't have friends in-game 😢

**Comment by quisquous** [comment]
This was fixed by #3167.
